RenderSystems/GL3Plus/CMakeLists.txt

```diff
@@ -13,24 +13,22 @@ file(GLOB HEADER_FILES "${CMAKE_CURRENT_SOURCE_DIR}/include/*.h")
 file(GLOB SOURCE_FILES "${CMAKE_CURRENT_SOURCE_DIR}/src/*.cpp")
 list(APPEND SOURCE_FILES "${CMAKE_CURRENT_SOURCE_DIR}/src/gl3w.c")
 
-file(GLOB GLSL_HEADERS "${CMAKE_CURRENT_SOURCE_DIR}/include/GLSL/*.h")
-file(GLOB GLSL_SOURCE "${CMAKE_CURRENT_SOURCE_DIR}/src/GLSL/*.cpp")
-
-set(HEADER_FILES ${HEADER_FILES} ${GLSL_HEADERS})
+file(GLOB GLSL_HEADERS "${CMAKE_CURRENT_SOURCE_DIR}/src/GLSL/include/*.h")
+file(GLOB GLSL_SOURCE "${CMAKE_CURRENT_SOURCE_DIR}/src/GLSL/src/*.cpp")
 
 add_definitions(${OGRE_VISIBILITY_FLAGS})
 
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/src/GLSL/include)
```

Comment:
please keep this at the target_include_directories call below as:
```sh
    "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src/GLSL/include>"
```

Reply:
My mistake, should it be `PRIVATE` too?

Reply:
yes, please